runtime-infra/fragments/sqs-queue.yaml

```diff
@@ -263,6 +267,30 @@ Resources:
       OKActions:
         - !Sub 'arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${AlarmSNSTopicName}'
 
+  OncallDlqOldMessagesAlarm:
```

Comment:
@mbrachi devi creare un allarme simile a questo con la condizione `IsWithAlarm`. Vedi come esempio la risorsa `QueueHasMessagesAlarm`. In pratica serve se il frammento è usato per creare direttamente una DLQ invece che una coda standard + la sua DLQ.